WcaOnRails/app/views/registrations/edit_registrations.html.erb

```diff
@@ -42,7 +44,7 @@
 
         <tbody>
           <% registrations.each do |registration| %>
-            <tr id="registration-<%= registration.id %>" class="<%= registration.pending? ? "registration-pending" : "" %> <%= registration.accepted? ? "registration-accepted" : "" %>">
+            <tr id="registration-<%= registration.id %>" class="<%= registration.pending? ? "registration-pending" : "" %><%= registration.accepted? ? "registration-accepted" : "" %><%= registration.deleted? ? "registration-deleted" : "" %>">
```

Comment:
I think this might be cleaner with a hashmap. Something like this:

```ruby
registration-<%= { a: "accepted", p: "pending", d: "deleted" }[registration.checked_status] %>
```

I like how this makes it obvious that only one of these 3 states will happen at once.

Furthermore, if we decide to get rid of this `:a, :p, :d` stuff, then this could become as simple as:

```ruby
registration-<%= registration.checked_status %>
```